pkg/changeset/testdata/noncommitted/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/non_committed_branch
```

Comment:
We should confirm that we get a commit sha the way we run our releases.

Reply:
For release-0.1 and release-0.2, I verified we got a commit sha to .git/HEAD. What else should we do during release? A verification?